src/test/beatmaptest.cpp

```diff
@@ -62,6 +62,12 @@ TEST_F(BeatMapTest, Scale) {
 
     pMap->scale(Beats::THREEFOURTHS);
     EXPECT_DOUBLE_EQ(bpm / 2, pMap->getBpm());
+
+    pMap->scale(Beats::FOURTHIRDS);
```

Comment:
Oookay, think I got it! Just to bes sure:
TEST(BeatGridTest, Scale) is (like?) and function and called, therein all pGrid->scale(Beats::XXXX) are applied one after another, return values are checked in between.